packages/opencensus-propagation-jaeger/src/jaeger-format.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {HeaderGetter, HeaderSetter, Propagation, SpanContext} from '@opencensus/core';
+
+import * as crypto from 'crypto';
+import * as uuid from 'uuid';
+
+const UBER_TRACE_ID = 'uber-trace-id';
```

Comment:
Is it more accurate to call this the "uber Trace ID" or the "Jaeger Trace ID"?

Reply:
We can't really change the header value, as the client libraries all assume uber-trace-id. Dropping the prefix seems reasonable.

Reply:
Thanks. I was referring to the variable name only.